org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/extensions/InjectionExtension.java

```diff
@@ -28,6 +30,9 @@
  * The extension retrieves an {@link Injector} to inject members in the instance
  * under test before best execution and resets registries after execution.
  * </p>
+ * It takes care about JUnit5 @Nested test classes. That are inner classes that might
```

Comment:
This should be `{@link Nested nested} test classes` - could be, that the JavaDoc validation will fail if we do it like is is done right now.

Reply:
done